src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.util.ArrayList;
+import duke.io.Storage;
+import duke.task.TaskList;
+import duke.io.Ui;
+import duke.io.Layout;
+import duke.task.DukeException;
```

Comment:
Maybe you could order the imports by their type? i.e. all the imports from duke.io together and duke.task together